helm_deploy/manage-my-prison/values.yaml

```diff
@@ -13,6 +15,9 @@ generic-service:
     tag: app_version    # override at deployment time
     port: 3000
 
+  service:
+    enabled: true
```

Comment:
I tried to think of a way of doing this but I can't think of anything at the moment.

If we had more control over the normal `Service` we could just change the `selector` conditionally when in maintenance mode (to point to the maintenance pods instead of the app pods, what you do here).

But this `Service` comes [courtesy of `generic-service` subchart](https://github.com/ministryofjustice/hmpps-helm-charts/blob/324fb40a901a8fcd468448dce71d1b6f142897a2/charts/generic-service/templates/service.yaml) so I guess this is the next simpler way 👍

Reply:
yeah that's right, the maintenance mode service needs some parameters that we cannot override from the generic-service subchart 🤷